string mode             # sets randon either on, off or rgb
float64[] rgb_color     # colors
---
bool resp
